internal/querycoordv2/server.go

```diff
@@ -648,28 +648,32 @@ func (s *Server) handleNodeUp(node int64) {
 		zap.String("resourceGroup", rgName),
 	)
 
-	for _, collection := range s.meta.CollectionManager.GetAll() {
-		log := log.With(zap.Int64("collectionID", collection))
-		replica := s.meta.ReplicaManager.GetByCollectionAndNode(collection, node)
-		if replica == nil {
-			replicas := s.meta.ReplicaManager.GetByCollectionAndRG(collection, rgName)
-			if len(replicas) == 0 {
-				continue
-			}
-			sort.Slice(replicas, func(i, j int) bool {
-				return replicas[i].Len() < replicas[j].Len()
-			})
-			replica := replicas[0]
-			// TODO(yah01): this may fail, need a component to check whether a node is assigned
-			err = s.meta.ReplicaManager.AddNode(replica.GetID(), node)
-			if err != nil {
-				log.Warn("failed to assign node to replicas",
-					zap.Int64("replicaID", replica.GetID()),
-					zap.Error(err),
-				)
+	rgs := s.meta.ResourceManager.ListResourceGroups()
+	if len(rgs) == 1 {
```

Comment:
Move these to resource manager may make sense

Reply:
> Move these to resource manager may make sense

resource manager doesn't care about how to assign node to replica